stage6/01-install-packages/00-packages

```diff
@@ -30,7 +30,6 @@ doxygen
 libfftw3-dev
 libsdl1.2-dev
 libgsl-dev
-libqwt-qt5-dev
```

Comment:
maybe let's create an internal ticket to re-use the default/packaged qwt library;
the one that scopy uses looks in pretty bad shape;